# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/MediaCodecVideoCodec.h
# Commit: 9d8f6e354d8c
# Full Hash: 9d8f6e354d8c332bab7e433252077ec0e1fa8f04
# Author: John Lin <jolin@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - remove MediaCodecVP8Codec. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D115531
# ==============================================================================

diff -r 7675ead797c8 -r 9d8f6e354d8c dom/media/webrtc/libwebrtcglue/MediaCodecVideoCodec.h
--- a/dom/media/webrtc/libwebrtcglue/MediaCodecVideoCodec.h	Fri Oct 08 15:57:53 2021 +0200
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,31 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef MediaCodecVideoCodec_h__
-#define MediaCodecVideoCodec_h__
-
-#include "MediaConduitInterface.h"
-
-namespace mozilla {
-class MediaCodecVideoCodec {
- public:
-  enum CodecType {
-    CODEC_VP8,
-  };
-  /**
-   * Create encoder object for codec type |aCodecType|. Return |nullptr| when
-   * failed.
-   */
-  static WebrtcVideoEncoder* CreateEncoder(CodecType aCodecType);
-
-  /**
-   * Create decoder object for codec type |aCodecType|. Return |nullptr| when
-   * failed.
-   */
-  static WebrtcVideoDecoder* CreateDecoder(CodecType aCodecType);
-};
-
-}  // namespace mozilla
-
-#endif  // MediaCodecVideoCodec_h__